fendermint/app/src/cmd/run.rs

```diff
@@ -21,6 +23,15 @@ cmd! {
   }
 }
 
+async fn create_parent_finality(_settings: &Settings) -> anyhow::Result<InMemoryFinalityProvider> {
+    // let provider = InMemoryFinalityProvider::new(
+    //     settings.parent_finality.clone(),
+    //     unimplemented!(),
+    // );
+    // Ok(provider)
+    todo!()
+}
```

Comment:
Maybe you could have a default implementation which does nothing, for the case when the node is the root?

Reply:
yep, it's added